packages/api-client/src/views/Request/RequestSidebar.vue

```diff
@@ -218,6 +218,26 @@ const toggleSearch = () => {
   // Simply toggle the visibility
   isSearchVisible.value = !isSearchVisible.value
 }
+
+// Get started state is visible if
+// - only one request
+// - that request is in the drafts collection
+// - that request is not renamed
+const isGettingStarted = computed(() => {
+  const draftCollection = activeWorkspaceCollections.value.find(
+    (collection) => collection.info?.title === 'Drafts',
+  )
+  const hasSingleRequest = activeWorkspaceRequests.value.length === 1
+  const isDraftsRequest = draftCollection?.requests.includes(
+    activeWorkspaceRequests.value[0] ?? '',
+  )
+
+  if (!isDraftsRequest) return false
+  const isRenamed =
+    requests[draftCollection?.requests[0] ?? '']?.summary !== 'My First Request'
+
+  return hasSingleRequest && isDraftsRequest && !isRenamed
+})
```

Comment:
We could make this a separate file and tests for the cases. Might be annoying to mock up the state, maybe we use the opportunity to add test helpers or something.

Reply:
💯

Reply:
all logic should be in a helper lib, tested, then consumed in the computed